README.md

```diff
@@ -1,44 +1,104 @@
 # sofria-cli
+
 Command Line wrapper for Proskomma Sofria
 
 # Install dependencies
+
 ```shell
 npm install
 ```
 
 # E.g
+
 ```shell
 node sofria_mediaId.js Abidji_N2ABIWBT_USX
 ```
 
-# test
+# test sofria_mediaId
+
 Use two test cases:
 
 - Spanish_N2SPNTLA_USX
+
 ```shell
 # clean output folder:
 rm -rf output/Spanish_N2SPNTLA_USX-json/*.*
 
 # run sofria cli
-node sofria_mediaId.js test/input/Spanish_N2SPNTLA_USX/ ./output Spanish_N2SPNTLA_USX
+node biblebrain_uploader.js run test/input/Spanish_N2SPNTLA_USX/ --generate-json=./output/Spanish_N2SPNTLA_USX-json
 ```
+
 The above test should create 150 .json files
 
 ```shell
-ls -1 output/Spanish_N2SPNTLA_USX-json/ |  wc -l 
+ls -1 output/Spanish_N2SPNTLA_USX-json/ |  wc -l
 ```
 
-
 - Akawaio_N2AKEBSS_USX
+
 ```shell
 # clean output folder:
 rm -rf output/Akawaio_N2AKEBSS_USX-json/*.*
 
 # run sofria cli
-node sofria_mediaId.js test/input/Akawaio_N2AKEBSS_USX/ ./output Akawaio_N2AKEBSS_USX
+node biblebrain_uploader.js run test/input/Akawaio_N2AKEBSS_USX/ --generate-json=./output/Akawaio_N2AKEBSS_USX-json
 ```
+
 The above test should create 28 .json files
 
 ```shell
-ls -1 output/Akawaio_N2AKEBSS_USX-json/ |  wc -l 
+ls -1 output/Akawaio_N2AKEBSS_USX-json/ |  wc -l
+```
+
+# test verses
+
+```shell
+# clean JSON output folder:
+rm -rf output/Akawaio_N2AKEBSS_USX-json/*.*
+```
+
+```shell
+# clean output folder:
+rm -rf ./output/Akawaio_N2AKEBSS_USX-db
+```
+
+```shell
+# create output folder:
+mkdir ./output/Akawaio_N2AKEBSS_USX-db
+```
+
+```shell
+# create db file:
+touch ./output/Akawaio_N2AKEBSS_USX-db/Akawaio_N2AKEBSS_USX.db
+```
+
+```shell
+node biblebrain_uploader.js run test/input/Akawaio_N2AKEBSS_USX/ --populate-db=./output/Akawaio_N2AKEBSS_USX-db/Akawaio_N2AKEBSS_USX.db --generate-json=./output/Akawaio_N2AKEBSS_USX-json
+```
+
+- Outcome
+
+```shell
+Generate JSON filese - Start process..
+Populate DB - Start process..
+Complete:  MAꞌSIU
+success
+File list (1) processing completed
+Complete USX File: test/input/Akawaio_N2AKEBSS_USX/040MAT.usx
+load verses success, rowcount 1069
+load tableContents success, rowcount 1
+```
+
+The above test should create 28 .json files as well
+
+```shell
+ls -1 output/Akawaio_N2AKEBSS_USX-json/ |  wc -l
+```
+
```

Comment:
could you please add a note about the tables that are created in sqlite. 
Specifically document that tables tableContents and verses were created, and note the columns (eg tableContents has columns "code, heading, title, name, chapters" and verses has columns "reference, html") 

- check whether the column is book_code or code, and correct throughout. Also check "abbrev" and "html". Are they populated / needed?
- 
This comment from dbp-etl is pretty much what I am expecting here also.
This client will create and populate the tables: tableContents and verses
		#
		# The tableContents table will store data related to each book and the fields are:
		# tableContents(book_code, heading, title, name, abbrev, chapters)
		#
		# The verses table will store data related to each verse for each book and the fields are:
		# verses(reference, html)

Reply:
> could you please add a note about the tables that are created in sqlite. Specifically document that tables tableContents and verses were created, and note the columns (eg tableContents has columns "code, heading, title, name, chapters" and verses has columns "reference, html")
> 
>     * check whether the column is book_code or code, and correct throughout. Also check "abbrev" and "html". Are they populated / needed?
> 
> 
> This comment from dbp-etl is pretty much what I am expecting here also. This client will create and populate the tables: tableContents and verses # # The tableContents table will store data related to each book and the fields are: # tableContents(book_code, heading, title, name, abbrev, chapters) # # The verses table will store data related to each verse for each book and the fields are: # verses(reference, html)

@bradflood I have added more detail about the tables that are created in sqlite and all columns. As you can see in the following line:

https://github.com/faithcomesbyhearing/sofria-cli/pull/9/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R11

Also, it has removed the `abbrev` column because it is not necessary and it has renamed the column `html` to `text` because it is related to the verse text.

Finally, I have checked and the correct column name for the book code is: `code`